cshared.go

```diff
@@ -116,8 +130,15 @@ func FLBPluginInit(ptr unsafe.Pointer) int {
 	return input.FLB_OK
 }
 
+func min(a, b int) int {
```

Comment:
go 1.21 has a built-in min method.

Reply:
I was unable to find the actual documentation for that function in the documentation. I had heard about it, but without documentation I was not going to rely on it.

I am testing the plugin now without my version of `min` and will push the change once it passes.